dunst/dunst

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+"""
+A do-not-disturb button for muting Dunst notifications in i3 using i3blocks
+
+Mute is handled by passing 'DUNST_COMMAND_PAUSE' and 'DUNST_COMMAND_RESUME' to
+the notify-send script and the 'DUNST_MUTE' environment variable is set to keep
+track of the toggle.
+"""
+
+__author__ = "Jessey White-Cinis <j@cin.is>"
+__copyright__ = "Copyright (c) 2019 Jessey White-Cinis"
+__license__ = "MIT"
+__version__ = "1.0.0"
+
+import os
+import subprocess
+import json
+
+def mute_on():
+    '''Turns off dunst notifications'''
+    subprocess.run(["notify-send","DUNST_COMMAND_PAUSE"])
+    return {
+        "full_text":"<span font='Font Awesome 5 Free Solid' color='#BE616E'>\uf1f6</span>",
+        "DUNST_MUTE":"on"
+    }
+
+def mute_off():
+    '''Turns back on dunst notifications'''
+    subprocess.run(["notify-send","DUNST_COMMAND_RESUME"])
```

Comment:
Using `subprocess.run` without explicitly set `check` is not recommended.

Reply:
Added a new commit with check explicitly set.